lib/perl/Genome/Site/TGI/Extension/generate_lims_disk_unarchive.pl

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env lims-perl
+
+use strict;
+use warnings;
+
+use Data::Dumper;
+use Carp qw/confess/;
+
+use GSCApp;
+
+my $ii = GSC::IndexIllumina->get(analysis_id => $ARGV[0]);
```

Comment:
Should check if $ARGV[0..1] are defined. Since you've gone into GSCApp land, you could use App::GetOpt 😄